### ISSUE TITLE
bless: The 'openfolder' is not supported on Apple Silicon devices.

### ISSUE THREAD

**Issue by anoopvaidya** [LinusU/node-appdmg#239]
I have been using appdmg for a long time, and I just noticed the issue in Apple Silicon devices. Since gradually I am moving from Intel based to Apple Silicon (M1, M2 etc) macs the `appdmg` tools started giving this error.

Attached is the screenshot showing the appdmg version and the error.
I saw Issue#204, but could not understand how can I use it on our project.

Please help us with this.

<img width="918" alt="appdmg issue" src="https://github.com/LinusU/node-appdmg/assets/13703960/403f9a76-9579-474b-a21f-96820e584c59">

**Comment by GZixeT** [comment]
Quick solution:
1. Open file `/usr/local/lib/node_modules/appdmg/lib/appdmg.js`
2. Find and remove code

```
if (os.arch() !== 'arm64') {
args.push('--openfolder', global.temporaryMountPath)
}
```

3. Save file
4. Try make dmg again
5. Done

Quick solution:
1. Download `master` appdmg
2.  Open folder `/usr/local/lib/node_modules/appdmg/lib/`
3. Replace files from `/usr/local/lib/node_modules/appdmg/` with files from [1]
4.  The folder `node_modules `should remain in `/usr/local/lib/node_modules/appdmg/`
5. Done

**Comment by anoopvaidya** [comment]
@GZixeT : I did both the steps but no luck. And one more thing the path you mentioned `/usr/local/lib/node_modules/appdmg/lib/appdmg.js` does not exist on my(teams) system. In fact the path is `/opt/homebrew/lib/node_modules/appdmg/lib/appdmg.js` 

Is this ok, or we need to do something else to get it updated?

**Comment by GZixeT** [comment]
> Is this ok, or we need to do something else to get it updated?

It's normal that you have a different path, try to follow the steps to follow your path. If all else fails, then your problem is different from mine, and you will have to look for your own solutions.

**Comment by anoopvaidya** [comment]
@GZixeT : I did the same changes, but Logically commenting the `if` condition that will never get executed on an ARM (Apple silicon system), how can that fix the issue?

```
if (os.arch() !== 'arm64') {
    args.push('--openfolder', global.temporaryMountPath)
}
```

Either you have an Intel chipset or you did something else to fix it.

**Comment by GZixeT** [comment]
> Either you have an Intel chipset or you did something else to fix it.

Yes, it's strange, I have an M2, but the first solution helped me. Just try, you will always have time to change the changes back
![image](https://github.com/user-attachments/assets/be3dea0c-9e95-4074-b793-58f794be3736)
![image](https://github.com/user-attachments/assets/ff4506de-84c0-4410-8491-67ca13974e3c)
![image](https://github.com/user-attachments/assets/f2fcf736-d529-408c-b973-026453ab4262)
